src/main/java/dev/emi/trinkets/mixin/LivingEntityMixin.java

```diff
@@ -212,29 +229,19 @@ private void tick(CallbackInfo info) {
 				Set<TrinketInventory> inventoriesToSend = trinkets.getTrackingUpdates();
 
 				if (!contentUpdates.isEmpty() || !inventoriesToSend.isEmpty()) {
-					PacketByteBuf buf = PacketByteBufs.create();
-					buf.writeInt(entity.getId());
-					NbtCompound tag = new NbtCompound();
+					var map = new HashMap<String, NbtCompound>();
 
 					for (TrinketInventory trinketInventory : inventoriesToSend) {
-						tag.put(trinketInventory.getSlotType().getGroup() + "/" + trinketInventory.getSlotType().getName(), trinketInventory.getSyncTag());
+						map.put(trinketInventory.getSlotType().getGroup() + '/' + trinketInventory.getSlotType().getName(), trinketInventory.getSyncTag());
```

Comment:
Not that important but why the change from a string to a char here? I preferred the string.